Formula/llvm.rb

```diff
@@ -136,10 +138,6 @@ def install
       args << "-DFFI_LIBRARY_DIR=#{Formula["libffi"].opt_lib}"
     end
 
-    # The latest stage builds avoid the shims, and the build
-    # will target Penryn unless otherwise specified
-    ENV.append_to_cflags "-march=#{Hardware.oldest_cpu}" if Hardware::CPU.intel?
```

Comment:
Removed this because it turns out this messes with the `compiler-rt` build, which expects to be able to compile for other non-native arches.